src/core/ext/transport/binder/wire_format/binder_android.cc

```diff
@@ -194,6 +196,15 @@ int32_t WritableParcelAndroid::GetDataPosition() const {
   return AParcel_getDataPosition(parcel_);
 }
 
+int32_t WritableParcelAndroid::GetDataSize() const {
+  if (AParcel_getDataSize) {
+    return AParcel_getDataSize(parcel_);
+  } else {
+    gpr_log(GPR_INFO, "[Warning] AParcel_getDataSize is not available");
+    return 0;
```

Comment:
This returns 0 but `ReadableParcelAndroid::GetDataSize` return -1?

Reply:
Done. Both return 0.